packages/jaeger-ui-components/src/types/index.tsx

```diff
@@ -13,7 +13,10 @@
 // limitations under the License.
 
 export { TraceSpan, TraceResponse, Trace, TraceProcess, TraceKeyValuePair, TraceLink } from './trace';
-export { SpanBarOptions, SpanBarOptionsData } from '../settings/SpanBarSettings';
+export {
+  SpanBarOptions,
+  SpanBarOptionsData,
+} from '../../../../public/app/core/components/SpanBarSettings/SpanBarSettings';
```

Comment:
If SpanBarOptions are needed also in jeager-ui-components maybe they should live there?

Reply:
I guess it's in core because it's used by multiple datasources (Jaeger, Zipkin and Tempo) so maybe it's better to leave it there as it's not a component specific to Jaeger. Thoughts?

Reply:
So why are we importing it here just to export it? Are `SpanBarOptions` actually required anywhere inside the jaeger-ui package?

Reply:
It seems like `SpanBarOptions` and `SpanBarOptionsData` is something that has lived in jaeger-ui so I assume it can stay there and just be exported and used in `SpanBarSettings` component?